shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -61,12 +58,9 @@ public static void startInitialization(@NonNull Context applicationContext) {
    */
   public static void startInitialization(
       @NonNull Context applicationContext, @NonNull Settings settings) {
-    if (isRunningInRobolectricTest) {
-      return;
-    }
     FlutterLoader.Settings newSettings = new FlutterLoader.Settings();
     newSettings.setLogTag(settings.getLogTag());
-    FlutterLoader.getInstance().startInitialization(applicationContext, newSettings);
+    FlutterInjector.instance().flutterLoader().startInitialization(applicationContext, newSettings);
```

Comment:
Might be less verbose to expose helper functions to fetch loader from the injector. Sort of like how modules work in DI frameworks.

Reply:
sure, it's a good idea. I'm currently leaning towards leaving things as simple as possible rather than having a production DI system since it's mostly for us and I want to keep it with as few indirections as possible for now.